Java/JDI/jdi-cucumber-test-generator/pom.xml

```diff
@@ -66,7 +71,13 @@
         <dependency>
             <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
-            <version>3.1.0</version>
+            <version>${servlet-api.version}</version>
```

Comment:
Once again I suggest to introduce dependency management section in parent pom and store all out version there.

Reply:
I added commit to the Epmlstrtai 40.